test/bundler/webpack/bundler-test.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const fastifySuccess = require('./dist/success')
+const fastifyFailPlugin = require('./dist/failPlugin')
+
+test('Bundler should work', t => {
+  t.plan(1)
+  fastifySuccess.ready((err) => {
+    t.error(err)
+  })
+})
+
+// Untill there is no proper solution for bundlers, the fastify version
+// is set to undefined for this context
+test('Bundler should work with bad plugin version, undefined version', t => {
```

Comment:
Is this test going to pass or not? I'm not sure it would. Maybe we should skip it in case - I want all the CI to stay green.

Reply:
So theorically on a non bundled environment it would fail, but on bundled environment because of the fallback in here:
https://github.com/fastify/fastify/blob/a90c68b4cd07318610ae59a8a60c0435c9cc881c/fastify.js#L688 it's set to undefined. So it passes.

Reply:
Can you clarify the comment so that it's clear what we should change here?

Reply:
Sure